Extension/src/LanguageServer/settingsPanel.ts

```diff
@@ -165,30 +166,37 @@ export class SettingsPanel {
         switch (message.key) {
             case elementId.activeConfig:
                 this.configValues.name = message.value;
+                telemetry.logLanguageServerEvent("ConfigUI", { "ValueChanged": "configName" }, null);

```

Comment:
How much telemetry is this going to send?  Is there a way to tell when the user has stopped interacting with the UI (e.g. focus a different tab) and send some cached data instead?  Kind of like the way we batch up LSP messages when logging telemetry.

Reply:
The data can be cached and sent whenever the webview panel is disposed (either when the config UI window was closed or VS code was closed).